CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+# Contributing
+
+## General Workflow
+
+1. Fork the repo
+1. Cut a namespaced feature branch from master
+1. Make commits to your feature branch. Prefix each commit like so:
+1. When you've finished with your fix or feature, merge upstream changes into your branch. submit a pull request directly to master. Include a description of your changes.
```

Comment:
Are the pull requests just going to be open while someone is working on it? Is it for us to know what each individual of the team is working on?